### ISSUE TITLE
VPAT — Please Add Keyboard Functionality To Mouse Dependent Elements

### ISSUE THREAD

**Issue by pbailie** [Submitty/Submitty#4909]
Proper accessibility requires that all interactive elements are "device independent".  In short, this means that interaction cannot be dependent on the mouse alone.

While major browsers and assistive technologies may interpret interactive links and controls with an `[ENTER]` keypress by default, this is not to be universally relied upon.

I believe that `onclick` event handlers or similar mouse-only interaction should have an associated keyboard interaction.  One possibility is to implement `onkeypress` events that also checks for the `[ENTER]` and `[SPACE]` keys.

When reviewing PR #4882, I observed several instances of `onclick` event handlers with no keyboard equivalents.  I also observed instances where an `aria-label` was being added to a font icon to dictate "drag to reorder" or similar.

`grep`ping through twig and javascript code for `onclick`, `drag`, `click` (or other mouse related keywords) may be a good place to start this examination.

**Comment by elihschiff** [comment]
For anyone looking at this issue in the future, I think the only thing left to do is to fix draggable elements to work with the keyboard. Normal onclick elements have all been fixed.

**Comment by bmcutler** [comment]
Since we've switched to the new TA grading interface, hopefully this issue is closed.
If there are remaining items, let's open new/more specific issues.
